### DISCUSSION TITLE
Allure Report Customization - Java Junit 5 test framework

### DISCUSSION THREAD

**Discussion by sumanthvaka** [allure-framework/allure2#2412] [General Discussion]
Hello,

I would like to modify the test cases count on summary widget in allure report, want to include the step count as well in the overview. That customization is possible? I am using allure java - JUnit5 test framework. Can you please help me with report customization.

Thanks

**Comment by baev** [comment]
It is possible to create a custom plugin for Allure Report, but as for today, it would be much easier to fork it and do the required fixes. However, if you explain the need in more detail, maybe we can suggest you a solution (or even consider implementing such feature)

**Reply by sumanthvaka** [reply]
Thank you for replying. I am actually looking to add the step counts to the no of test cases count I mean( Total count: Test cases count + step count) so I would like to to change current showing total test cases count in overview dashboard in the report. I would like to include step count as well. I saw summary.json has total variable that actually reflects in report. Is there a way that I can access summary.json to change the report statistics or any TestResults method that can handle the report modifications? Helping on modifying test cases count through JAVA  would be appreciated.

Thanks

**Reply by sumanthvaka** [reply]
Is that possible? Can you please let me know how can I achieve this scenario
